lib/textToImage.js

```diff
@@ -25,21 +26,27 @@ const createTextData = (
   fontFamily,
   fontWeight,
   customHeight,
+  textAlign,
 ) => {
   // create a tall context so we definitely can fit all text
   const textCanvas = Canvas.createCanvas(maxWidth, 1000);
   const textContext = textCanvas.getContext('2d');
-  const textX = 0;
+  var textX = 0;
+  if ( textAlign.toLowerCase() == "center" ) textX = maxWidth / 2;
+  if ( textAlign.toLowerCase() == "start" || textAlign.toLowerCase() == "left" ) textX = 0;
+  if ( textAlign.toLowerCase() == "end" || textAlign.toLowerCase() == "right" ) textX = maxWidth;
```

Comment:
Will this align the left edge of the text with the right side of the canvas? I.e. rendering the text completely outside the canvas? 🤔